directives/meta.py

```diff
@@ -1,8 +1,21 @@
+import copy, re
 from docutils import nodes
 from docutils.parsers.rst import Directive, directives
+from sphinx.errors import SphinxError
 
 from aplus_nodes import aplusmeta
 
+def setup(app):
+    # The AplusMeta directive is already set in aplus_setup.py:setup(app).
+    # However, if one wants to use the `aplusmeta_substitutions` variable in conf.py,
```

Comment:
Isn't `aplusmeta_substitutions`already registered in the `aplus_setup`, so adding this file to the extensions, shouldn't have any effect? Basically, Sphinx will call the setup from modules listed in the extensions, hence this empty setup doesn't do anything, thus there is no point of adding it to the extensions. Am I right?

Reply:
I get the same impression that this empty `setup` function is meaningless and that there is no need to add `meta` to the `extensions` of `conf.py`. The new variable `aplusmeta_substitutions` shall be part of the core `aplus_setup` extension. It shouldn't do any harm even if somebody doesn't want to use it. I don't see the benefit in hiding it behind a tiny extra extension. Like raphendyr said, the variable is already defined in `aplus_setup.py`, thus this empty `setup` can be removed.

Reply:
True, I re-checked that `meta` can be removed from `extensions` in `conf.py`.